SAP/spartacus
# chore: Inline release notes scripts

QA steps:
- add github token env to your shell `export GITHUB_TOKEN=whatever`
- run some release scripts eg. `npm run release:assets:with-changelog`
- select version bump (eg. 3.0.0)
- check that after building (before commit step) it outputs release notes for that library and for this version (it should be exactly the same as running the command on different branch with the same library and version)
- if everything is the same then this PR does not brake anything, but clears unnecessary scripts from `package.json`
